### DISCUSSION TITLE
Get outputted voltage

### DISCUSSION THREAD

**Discussion by Did-Mas** [ni/nidaqmx-python#658] [Q&A] [answered]
Is there a way to check the current voltage being output on a given channel? For example, if a voltage was set on a channel but never reset to 0, I would like to be able to check the previously set voltage before setting a new value.

**Comment by zhindes** [comment, marked as answer]
No, there is no native way to do that with an Analog Output channel in NI-DAQmx. You need to save off any voltage values you have written if they are relevant later.
